server/src/main/proto/spine/server/delivery/delivery.proto

```diff
@@ -107,3 +108,45 @@ message WorkerId {
     // An example of such an identifier could be ID of the thread which processes a shard.
     string value = 2 [(required) = true];
 }
+
+// One of the possible outcomes of shard pick-up request stating that the shard is already picked
+// by another worker.
+message ShardAlreadyPickedUp {
+
+    // A worker that owns the session.
+    WorkerId worker = 1 [(required) = true];
+
+    // Time when the `worker` has picked the shard.
```

Comment:
"picked up".

It's everywhere, I think.

"To pick" is "to choose". "To pick up" is closer to "grab and possess".